move-to-bugzilla.js

```diff
@@ -175,6 +175,12 @@ async function populateMoveForm(sections, pathname) {
   });
   moveButton.disabled = false;
 
+  const resetButton = document.getElementById("reset-move-form");
+  resetButton.addEventListener("click", () => {
+    section.storage.clear();
+    populateMoveForm(sections, pathname);
```

Comment:
This probably doesn't quite work because you end up creating new `controls` wrappers and rebinding event listeners. I I think you probably want `populateFromIssue(controls, issue)`

Reply:
Oh interesting. It worked for me, but you're right, that makes more sense. Fixed.